sftp-client.c

```diff
@@ -1193,6 +1222,41 @@ do_download(struct sftp_conn *conn, const char *remote_path,
 
 	attrib_clear(&junk); /* Send empty attributes */
 
+#ifdef IRONCORE
+	if (strlen(local_path) > PATH_MAX - IRON_SECURE_FILE_SUFFIX_LEN) {
+		error("Local path name %s too long.", local_path);
+		return(-1);
+	}
+	const char * output_path;
+	char iron_name[PATH_MAX + IRON_SECURE_FILE_SUFFIX_LEN + 1];
```

Comment:
Is it a good idea to create paths which are longer than `PATH_MAX`? I suspect other apps and/or the OS will be unhappy about that.